### PR TITLE
Updates to roadmap

### PR DESCRIPTION
**Pull request by shebuk**
Changes to reflect completed work and plans for CQ2'23

### CODE REVIEW

File: project/roadmap.md [hunk 1]
```diff
@@ -4,61 +4,71 @@
...
- * [Mbed TLS] [Code size optimisation (driver only build - ECC)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [Mbed TLS] [Code size optimisation (driver only build - ECC)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [Mbed TLS] [TLS1.3 early data](https://github.com/orgs/Mbed-TLS/projects/1#column-18972084) 
+* [PSA Crypto] Memory Optimizations
+* PAKE API investigation for SPAKE2+ 
+* [PSA Crypto] [PBKDF2 PSA Driver Implementation](https://github.com/orgs/Mbed-TLS/projects/1#column-18986190)
+
+**2023 CQ4**
+* [Mbed TLS] [Code size optimisation (driver only build - Cipher and AEAD)](https://github.com/orgs/Mbed-TLS/projects/1#column-19075367)
+* [PSA Crypto] PSA driver – Handle Opaque Persistent Key in Secure Element - Investigation
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:24]: This line is repeated from Q2. Should it be removed, or should this be a different work item?

File: project/roadmap.md [hunk 2]
```diff
@@ -4,61 +4,71 @@
...
-* [Mbed TLS] [Mbed TLS3.2](https://github.com/orgs/Mbed-TLS/projects/1#column-18338314)
-* [Mbed TLS] [TLS1.3 PSK](https://github.com/orgs/Mbed-TLS/projects/1#column-17950145)
-* [Mbed TLS] [TLS/X.509​ - Use PSA Long term secret isolation](https://github.com/orgs/Mbed-TLS/projects/1#column-183383222)
-* [Mbed TLS] [Use PSA: code size optimisation via PSA_CRYPTO_CONFIG - part1: symmetric](https://github.com/orgs/Mbed-TLS/projects/1#column-18883163)
-* [Mbed TLS] [Connection ID DTLS1.2](https://github.com/Mbed-TLS/mbedtls/pull/6264)
-
-**2023 CQ1 (In Development)**
+**2023 CQ2 (In Development)**
+* [Mbed TLS] [Code size optimisation (driver only build - ECC - Initial)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [PSA Crypto] Memory Optimizations - Phase1
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:9]: When I see “memory optimizations” out of context, I think of RAM optimization. But our focus is on code size, so I think this and other similar lines should say “code size optimizations”.

**Review by gilles-peskine-arm — CHANGES_REQUESTED** [review]
There seems to be an editing mistake

File: project/roadmap.md (hunk 2 above)
Comment (shebuk) [reply project/roadmap.md]: Added (code size) for the Memory optimization entries

**Review by gilles-peskine-arm — APPROVED** [review]
LGTM

**Pull request merged**